cmd/pod-scaler/admission.go

```diff
@@ -0,0 +1,211 @@
+package main
```

Comment:
I am not a fan of handrolling stuff we could just use a lib for like e.G. in https://github.com/kubernetes-sigs/controller-runtime/blob/master/examples/builtins/mutatingwebhook.go